akka-actor-testkit-typed/src/test/java/jdocs/akka/actor/testkit/typed/javadsl/TestConfigExample.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package jdocs.akka.actor.testkit.typed.javadsl;
+
+// #default-application-conf
+import com.typesafe.config.ConfigFactory;
+
+// #default-application-conf
+
+public class TestConfigExample {
+
+  void illustrateApplicationConfig() {
+
+    // #default-application-conf
+    ConfigFactory.defaultApplication()
```

Comment:
When testing against `application.conf`, I guess what people are doing is testing 'the configuration as it is used when running'. Wouldn't you usually want to use `ConfigFactory.load()` in that case then (so also loading `reference.conf` defaults)?

Reply:
That's right, better to use `ConfigFactory.load(), which loads application.conf

We do the sandwiching load in the ActorSystem, but makes sense to use the full load here as well.